Src/CSharpier/DocPrinter.cs

```diff
@@ -63,11 +63,11 @@ private Indent MakeIndent(Indent indent, Options options)
                             AddSpaces(options.TabWidth);
                         }
                         break;
-                    case "stringAlign":
-                        Flush();
-                        value += part.Number;
-                        // TODO 2 huh? length += part.n.length;
-                        break;
+                    // case "stringAlign":
```

Comment:
Did you forget about this, or are you leaving it as a todo reminder? 
If it's the latter, I think you're better of just having a TODO comment. Commented-out-code just isn't very useful with revision control.